package.json

```diff
@@ -8,6 +8,7 @@
     "test": "vitest"
   },
   "devDependencies": {
+    "@babel/core": "^7.23.9",
```

Comment:
@vstefanovic97 why is this needed in the root of monorepo? I guess each package that needs it should declare such kind of dependency.

Reply:
It's used in tests [here](https://github.com/ember-cli-code-coverage/ember-cli-code-coverage/pull/405/files#diff-11851204535d7735df82299a725e271fc96f8c040a5468c6070cba4a7ce479f6R2)

Reply:
ah, now I see - `test-packages` folder contains both vite tests *and* test packages.
IMO seems confusing but for sure we should follow existing structure in this PR.